src/hooks/useFirestoreCollectionWithAtom.ts

```diff
@@ -405,6 +405,13 @@ export const tableFiltersToFirestoreFilters = (filters: TableFilter[]) => {
         where(filter.key.concat(".hex"), "!=", filter.value.hex.toString())
       );
       continue;
+    } else if (filter.operator === "array-contains") {
+      if (!filter.value || !filter.value.length) continue;
+      // make the value as a singular string
+      firestoreFilters.push(
+        where(filter.key, filter.operator as WhereFilterOp, filter.value[0])
+      );
+      continue;
```

Comment:
I understand why you had to do `filter.value[0]` and not use `filter.value`.

As a side effect, this has created bad user experience. Check the video below.

https://github.com/rowyio/rowy/assets/4337699/8afc6f68-e695-48d6-95c1-ac8ad4e60b21

I expected only 1 row with `["ABC", "XYZ"] because this is what *"contains the following"* mean.

To help with the UX we can disable the "add short text" button and not allow the user to add more than one field. This should be doable by checking in the operator prop in ArraySubDrawerField. Disable the button if the operator is 'array-contains" and one entry has already been added.